hraven-core/src/main/java/com/twitter/hraven/rest/ObjectMapperProvider.java

```diff
@@ -108,13 +117,155 @@ public void serialize(Configuration conf, JsonGenerator jsonGenerator,
   }
 
   /**
+   * Custom serializer for TaskDetails object.
+   */
+  public static class TaskDetailsSerializer extends JsonSerializer<TaskDetails> {
```

Comment:
The other serializer classes have been public, hence continued to have these as public classes. If I make the JobDetailsSerializer class, then the hRavenRestClient class complains at line 504 https://github.com/twitter/hraven/pull/141/files#diff-d6dfbdd207a717584e1a98b1e528aefdR504